axelor-project/src/main/resources/views/ProjectWizard.xml

```diff
@@ -9,6 +9,9 @@
       <field name="$project" title="Project" colSpan="12" type="many-to-one"
         onChange="action-project-record-empty-sprint,action-attrs-refresh-dashlets"
         target="com.axelor.apps.project.db.Project"/>
+      <field name="employee" target="com.axelor.apps.hr.db.Employee" title="Employee"
+        type="many-to-one" colSpan="12"
+        domain="date(self.hireDate) &lt;= date(CURRENT_DATE()) and( date(self.leavingDate) &gt;= date(CURRENT_DATE())  or self.leavingDate is NULL )"/>
```

Comment:
I think the domain will be better in java, to exclude all employee linked to a user which is not in the project.memberUserSet.

And throw the refresh on this change too.